rwt_moveit/launch/sim_demo.launch

```diff
@@ -1,33 +1,38 @@
 <launch>
-<arg name="fixed_frame" default="/BASE" />
-<param name="sim_mode" value="true" />
-<param name="fixed_frame" value="$(arg fixed_frame)" />
-<node pkg="tf" type="static_transform_publisher" name="connect_start" args="0 0 0 0 0 0 $(arg fixed_frame) /start$(arg fixed_frame) 100" />
-<node pkg="tf" type="static_transform_publisher" name="connect_goal" args="0 0 0 0 0 0 $(arg fixed_frame) /goal$(arg fixed_frame) 100" />
-<node name="robot_state_publisher" pkg="robot_state_publisher" type="robot_state_publisher" />
-<node  name="start_state_publisher" pkg="robot_state_publisher" type="robot_state_publisher" >
-  <remap from="joint_states" to="start_joint_states" />
-  <param name="tf_prefix" type="string" value="start" />
-</node>
-<node  name="goal_state_publisher" pkg="robot_state_publisher" type="robot_state_publisher" >
-  <remap from="joint_states" to="goal_joint_states" />
-  <param name="tf_prefix" type="string" value="goal" />
-</node>
-<param name="use_gui" type="bool" value="true" />
-<node name="joint_state_publisher" pkg="rwt_moveit" type="joint_state_publisher" />
-<node name="virtual_joint_state_publisher" pkg="rwt_moveit" type="virtual_joint_state_publisher" />
-<node name="tf2_web_republisher" pkg="tf2_web_republisher" type="tf2_web_republisher" />
-<include file="$(find rosbridge_server)/launch/rosbridge_websocket.launch" />
+  <arg name="launch_roswww" default="true" />
```

Comment:
Same as https://github.com/tork-a/visualization_rwt/pull/50/files#r24293314 
I'm just wondering. So please explain the usecase.